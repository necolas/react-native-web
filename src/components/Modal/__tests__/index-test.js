/* eslint-env jasmine, jest */
describe('components/Modal', () => {
  test.skip('NO TEST COVERAGE', () => {});
});
