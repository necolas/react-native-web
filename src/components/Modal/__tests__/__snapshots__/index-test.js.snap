// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/Modal visible when "false" a container is not rendered 1`] = `null`;

exports[`components/Modal visible when "true" a container is rendered 1`] = `
<div
  className="rn-alignItems-1oszu61 rn-backgroundColor-1ikg2hk rn-borderTopStyle-1efd50x rn-borderRightStyle-14skgim rn-borderBottomStyle-rull8r rn-borderLeftStyle-mm0ijv rn-borderTopWidth-13yce4e rn-borderRightWidth-fnigne rn-borderBottomWidth-ndvcnb rn-borderLeftWidth-gxnn5r rn-bottom-1p0dtai rn-boxSizing-deolkf rn-color-homxoj rn-display-6koalj rn-flexShrink-1qe8dj5 rn-flexBasis-1mlwlqe rn-flexDirection-eqz5dr rn-font-1lw9tu2 rn-left-1d2f490 rn-listStyle-1ebb2ja rn-marginTop-1mnahxq rn-marginRight-61z16t rn-marginBottom-p1pxzi rn-marginLeft-11wrixw rn-minHeight-ifefl9 rn-minWidth-bcqeeo rn-paddingTop-wk8lta rn-paddingRight-9aemit rn-paddingBottom-1mdbw0j rn-paddingLeft-gy4na3 rn-position-1xcajam rn-right-zchlnj rn-textAlign-1ttztb7 rn-textDecoration-bauka4 rn-top-ipm5af rn-zIndex-sfbmgh"
/>
`;
